### ISSUE TITLE
[feature request] Support other cryptocurrencies like Zcash

### ISSUE THREAD

**Issue by heyakyra** [Voluntarynet/Bitmarkets#34]
Ideally, the market could be currency-agnostic. [Zcash](https://z.cash/) uses zero-knowledge security for privacy. There is a [Bitcoin API](https://z.cash/support/zig.html#bitcoin-api) for multisig transactions. While these transactions [don't yet have privacy](https://z.cash/support/faq.html#multisig-support) it is [being worked on](https://github.com/zcash/zcash/issues/782)

**Comment by stevedekorte** [comment]
Thanks for the suggestion. I'd like to support Monero in the new web version of Bitmarkets when they add multisig support and there is a javascript client (websocket gateways would be needed).

**Comment by zookozcash** [comment]
How about Zcash? You would need multisig (☑) and a Javascript client (☐)?

**Comment by stevedekorte** [comment]
Yeah, I'd be happy to support Zcash if there was an in-browser JS wallet library (with multi-sig support) for it that could connect to lots of independent Zcash nodes via websockets. Btw, there are lots of potential applications outside of Bitmarkets for in-browser JS integration.

**Comment by stevedekorte** [comment]
What's the status of a in-browser JS SPV wallet for zcash?
